external/sql/storm-sql-core/src/test/org/apache/storm/sql/compiler/backends/trident/TestPlanCompiler.java

```diff
@@ -274,4 +276,9 @@ private void waitForCompletion(long timeout, Callable<Boolean> cond) throws Exce
       Thread.sleep(100);
     }
   }
+
+  private void assertArrayWithoutOrder(List<Values> expected, List<List<Object>> actual) {
+    Assert.assertTrue("Two arrays are not same!\n"+ "Expected: " + expected + "\n" + "Actual: " + actual,
+            expected.containsAll(actual) && actual.containsAll(expected));
```

Comment:
@satishd Addressed via using CollectionUtils.isEqualCollection() in commons collections 4.

http://commons.apache.org/proper/commons-collections/javadocs/api-release/org/apache/commons/collections4/CollectionUtils.html#isEqualCollection(java.util.Collection,%20java.util.Collection)